documentation/source/building.rst

```diff
@@ -177,6 +180,10 @@ Regression Manager
     .. deprecated:: 1.5
         :class:`cocotb.hook` is deprecated, and in the future this variable will be ignored.
 
+.. envvar:: COCOTB_PDB_ON_EXCEPTION
+
+   If defined, cocotb will drop into the Python debugger (:mod:`pdb`) if a test fails with an exception.
```

Comment:
You mention later that this doesn't work on many simulators, maybe you should reiterate that here.

Reply:
Done.